src/components/Comments/Comment.js

```diff
@@ -52,7 +52,8 @@ const Comment = ({
     upvoteRemoved: null,
     downvoteInserted: null,
     downvoteRemoved: null
-  }
+  },
+  refreshComment
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `refreshComment` prop is a good change as it allows the component to refresh comments when needed. However, ensure that this prop is being passed from the parent component and has a default value or fallback in case it's not provided.
```diff
-  refreshComment
+  refreshComment = () => {}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->